pyanaconda/modules/network/initialization.py

```diff
@@ -130,6 +135,8 @@ def run(self):
                     ifname_option_values=self._ifname_option_values
                 )
 
+        mainctx.pop_thread_default()
```

Comment:
This won't be called if the code above fails with an exception.

Reply:
Addressed in https://github.com/rhinstaller/anaconda/pull/4322